### PR TITLE
695-update-resource-allocations

### PR DESCRIPTION
**Pull request by aprilrieger**
Refs https://github.com/scientist-softserv/dev-ops/issues/695

Please read the initial issue linked above

### CODE REVIEW

File: ops/production-deploy.tmpl.yaml [hunk 1]
```diff
@@ -5,11 +5,11 @@ imagePullSecrets:
 
 resources:
   limits:
-    memory: "4Gi"
```
Comment (orangewolf) [inline ops/production-deploy.tmpl.yaml:8]: this should be 2/4GB like other web processes. I can understand matching UTK, but we're not importing massively here so I think the worker can be much smaller as well

**Review by orangewolf — CHANGES_REQUESTED** [review]
I think pals prod can be a lot smaller

**Comment by aprilrieger** [comment]
@orangewolf I updated the values, let me know if these work.

**Review by orangewolf — APPROVED** [review]
looks great!

**Pull request merged**